### PR TITLE
Adding backup key server in install script

### PR DESCRIPTION
**Pull request by fkautz**

### CODE REVIEW

File: hack/install.sh [hunk 1]
```diff
@@ -414,7 +414,7 @@ do_install() {
 			fi
 			(
 			set -x
-			$sh_c "apt-key adv --keyserver hkp://pool.sks-keyservers.net:80 --recv-keys ${gpg_fingerprint}"
```
Comment (somepeopleskidz) [inline hack/install.sh]: Install$

Comment (somepeopleskidz) [inline hack/install.sh]: That$$$

**Comment by thaJeztah** [comment]
ping @tianon @jfrazelle ptal :+1: 

Thanks @fkautz!

File: hack/install.sh [hunk 2]
```diff
@@ -414,7 +414,7 @@ do_install() {
...
+			$sh_c "apt-key adv --keyserver hkp://pool.sks-keyservers.net:80 --recv-keys ${gpg_fingerprint} || apt-key adv --keyserver hkp://pgp.mit.edu:80 --recv-keys ${gpg_fingerprint}"
```
Comment (tianon) [inline hack/install.sh]: Does `pgp.mit.edu` support port 80 usage?  I've never actually checked. :innocent:

Comment (tianon) [inline hack/install.sh]: Neat, it does!

**Comment by tianon** [comment]
LGTM

**Comment by icecrime** [comment]
Thanks, LGTM

**Pull request merged**